task-driver/integration/tests/settle_match.rs

```diff
@@ -171,9 +174,18 @@ async fn setup_match_result(
         min_amount_order_index: false,
     };
 
-    // Simulate a reblind that would happen before a match
-    wallet1.reblind_wallet();
-    wallet2.reblind_wallet();
```

Comment:
confirming we can remove this b/c reblinding occurs in `setup_wallet_with_order_balance`, right?